awx/api/views/instance_install_bundle.py

```diff
@@ -51,6 +52,17 @@ class InstanceInstallBundle(GenericAPIView):
     permission_classes = (IsSystemAdminOrAuditor,)
 
     def get(self, request, *args, **kwargs):
+        receptor_installation_method = request.query_params.get('receptor_installation_method', settings.DEFAULT_RECEPTOR_INSTALLATION_METHOD)
+        if receptor_installation_method not in settings.RECEPTOR_INSTALLATION_METHODS:
+            return Response(
+                data=dict(msg=_(f"Invalid receptor installation method. Valid values are: {settings.RECEPTOR_INSTALLATION_METHODS}")),
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        # remove receptor_installation_method from query_params
+        request.query_params._mutable = True
```

Comment:
There's a better way to avoid the problem, I think, than reaching into and manipulating such an important object.

Reply:
let me try that